CMakeLists.txt

```diff
@@ -36,6 +36,10 @@ if(EXISTS "${TOP_DIRECTORY}/.git" AND NOT EXISTS "${TOP_DIRECTORY}/src/libhamcor
     message (FATAL_ERROR "Submodules are not initialized. Run\n\tgit submodule update --init --recursive")
 endif()
 
+if(WIN32 AND VCPKG_TARGET_TRIPLET AND NOT DEFINED CMAKE_TOOLCHAIN_FILE)
```

Comment:
what happens when user just open from VS without prior vcpkg initialization ? is `VCPKG_TARGET_TRIPLET` set ?
or do we fall under false condition ?

Reply:
Triplet is set in CMakeSettings. If vcpkg integration is not installed, VS won't set `CMAKE_TOOLCHAIN_FILE`. 
If a user creates new configurations without setting triplets, it is assumed vcpkg is not necessary and warning will not show.

Reply:
great, it comes from CMakeSettings

Reply:
I'm happy